starter_code/src/movies.js

```diff
@@ -1,15 +1,125 @@
 /* eslint no-restricted-globals: 'off' */
 
 // Iteration 1: Ordering by year - Order by year, ascending (in growing order)
+function orderByYear(array) {
+    let copyArray = array.slice();
+    return copyArray.sort(function (a, b) {
+        if (a.year > b.year) {
+            return 1;
+        } else {
+            return -1;
+        }
+    });
+}
+
+orderByYear(movies)
 
 // Iteration 2: Steven Spielberg. The best? - How many drama movies did STEVEN SPIELBERG direct
 
+// Didn't work out
+/*function howManyMovies(array1){
+let filteredMovies = array1.filter(function(drama){
+    if (array1.director === "Steven Spielberg" && array1.genre === "Drama"){
+        return true;
+    } else {
+        return false;
+    }
+})
+} 
+// Didn't work out
+function howManyMovies (newList){
+    if (movies.director.includes("Steven Spielberg") && movies.genre.includes("Drama") ) {
+        return true
+    } else {
+        return false
+    }
+    console.log (movies)
+}
+*/
+
+// IT IS ALIVE!
+function howManyMovies(arr) {
+    let filteredArray = arr.filter(function (obj) {
+
+            if (obj.director === "Steven Spielberg") && obj.genre.includes("Drama")) {
+            return true;
```

Comment:
you could use && operator here to write little bit cleaner code but its nothing wrong with what you wrote